config/bootstrap.php

```diff
@@ -32,8 +32,8 @@
 use Symfony\Component\Yaml\Yaml;
 
 $container_builder = new ContainerBuilder();
-$container = $container_builder->build();
-ServiceLocator::setServiceContainerInstance($container);
+$legacyContainer = $container_builder->build();
```

Comment:
in some tests, using the new PhpErrorHandler you will have a notice about it: this was this container who was called in set_parameters.php, producing a notice.